src/main/resources/assets/appliedenergistics2/lang/zh_CN.lang

```diff
@@ -241,13 +241,26 @@ gui.appliedenergistics2.Partitioned=分区
 gui.appliedenergistics2.PartitionedOre=矿典过滤卡
 gui.appliedenergistics2.Precise=精确匹配
 gui.appliedenergistics2.Fuzzy=模糊匹配
+gui.appliedenergistics2.Filter=过滤
 gui.appliedenergistics2.SmallFontCraft=合成
 gui.appliedenergistics2.LargeFontCraft=+
 gui.appliedenergistics2.Renamer=自定义名称: (回车保存设置)
 gui.appliedenergistics2.OreFilterLabel=矿典过滤卡
 gui.appliedenergistics2.HoldShiftForTooltip=按住Shift以查看拓展tooltip
 gui.appliedenergistics2.Nothing=无
 
+# GUI Colors - ARGB
+gui.color.appliedenergistics2.SearchboxFocused=6E000000
```

Comment:
Did you mean to include the colors in the Chinese translation file? This will probably cause weird issues with resource packs not working for Chinese players.